.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +24,9 @@
 <!-- Go over all points below, and after creating the PR, tick the checkboxes that apply. -->
 <!-- If you're unsure about any of these, don't hesitate to ask in the Help Contributors room linked above. We're here to help! -->
 - [ ] Tested changes locally.
-- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX
+- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line) 
```

Comment:
I think leaving the "closes" wording on this line would make sure this is still as unambiguous as possible.

Reply:
Unfortunately, I had to change the wording, because GitHub tends to catch it.